css/style.css

```diff
@@ -46,35 +38,165 @@ Developer @GA in NYC
 /*******************************************/
 
 body {
-     color: #000;
-     font-size: 12px;
+     color: #121212;
+     background-color: #f0efef;
+     width: 1160px;
```

Comment:
Never put a width on your <body> tag! If you need a container element, use a <div> or a <section>. This will cause issues in some browsers and would definitely interfere with any responsive design once we get to that section of the class.